src/core/Chip/Chip/Chip.md

```diff
@@ -7,24 +7,30 @@ const removeAction = () => {
 };
 
 const exampleRef = React.createRef();
-
+const chipStyle = {
+  display: 'inline-block',
+  marginRight: '10px',
+  marginBottom: '24px'
```

Comment:
Should we remove the bottom margin to keep the look more consistent. Or is there some specific test purpose for this?

Reply:
As discussed, this was to demonstrate the need to use margins by the library user. Margins are not provided per component.